src/lib/slide-toggle/slide-toggle.scss

```diff
@@ -169,3 +162,15 @@ $mat-slide-toggle-spacing: 8px !default;
   transition-property: background-color;
   transition-delay: 50ms;
 }
+
+// Ripple positioning for the slide-toggle. Moves the ripple container into the center of the thumb.
+.mat-slide-toggle-ripple {
+  position: absolute;
+  top: $mat-slide-toggle-thumb-size / 2 - $mat-slide-toggle-ripple-radius;
+  left: $mat-slide-toggle-thumb-size / 2 - $mat-slide-toggle-ripple-radius;
+  height: $mat-slide-toggle-ripple-radius * 2;
+  width: $mat-slide-toggle-ripple-radius * 2;
+  border-radius: 50%;
+  z-index: 1;
+  pointer-events: none;
+}
```

Comment:
@mmalerba Why is it useful to have a newline at the end of each file? Or is this just convention? Always wondered about that 😄